core/src/main/java/io/questdb/cairo/TableWriter.java

```diff
@@ -605,6 +606,8 @@ public void addIndex(CharSequence columnName, int indexValueBlockSize) {
     }
 
     public void addPhysicallyWrittenRows(long rows) {
+        // maybe not thread safe but hey it's just a metric
```

Comment:
Using a plain `long` field and a racy increment makes this metric unreliable. It's better to use `j.u.c.LongAdder` to make sure to accumulate all changes in the end number.

Reply:
It's unlikely to be racy, only happens when partition job finishes. I used `AtomicLong` since adder looks like an overkill

Reply:
Sounds good.